TFLite/src/Node.py

```diff
@@ -0,0 +1,34 @@
+# Node module to store operator attributes
+class Node:
+    def __init__(self, label, value = None):
```

Comment:
nit: A short comment on the semantics of each field might be helpful if it is not too time consuming.

Reply:
Added a link in comments for semantic meaning and operator support